test/box/tx_man.result

```diff
@@ -540,7 +540,7 @@ tx1:commit()
  | ...
 tx2:commit()
  | ---
- | - 
+ | - - {'error': 'Transaction has been aborted by conflict'}
```

Comment:
I think it will be good to explain why test result changed in tx_man.test.lua. For example, write a comment before tx2: commit (), which explains what the conflict is and why it occurs.